gazebo/rendering/ApplyWrenchVisual.cc

```diff
@@ -307,6 +308,7 @@ void ApplyWrenchVisual::SetTorque(const ignition::math::Vector3d &_torqueVector,
 ///////////////////////////////////////////////////
 void ApplyWrenchVisual::UpdateForceVisual()
 {
+  IGN_PROFILE("ApplyWrenchVisual::UpdateForceVisual");
```

Comment:
It would be good to be consistent

```suggestion
  IGN_PROFILE("rendering::ApplyWrenchVisual::UpdateForceVisual");
```